app/src/main/res/layout/choose_data_folder_dialog_entry.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/root"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="16dp"
+    android:clickable="true"
+    android:focusable="true">
```

Comment:
If you add `android:background="?attr/selectableItemBackground"` here, there is a ripple effect when touching the item. That also makes keyboard navigation easier because the focused item is highlighted.